src/pal/src/thread/thread.cpp

```diff
@@ -21,6 +21,8 @@ Module Name:
 #include "pal/dbgmsg.h"
 SET_DEFAULT_DEBUG_CHANNEL(THREAD); // some headers have code with asserts, so do this first
 
+#include "stdmacros.h"
```

Comment:
Do you still need it to be included here with the most recent changes? I would rather delete this `#include` than try to make it compile. It is generally a bad idea to include headers from src\inc in the PAL.